closure/goog/math/integer.js

```diff
@@ -253,9 +253,10 @@ goog.math.Integer.prototype.toString = function(opt_radix) {
     return '-' + this.negate().toString(radix);
   }
 
-  // Do several (6) digits each time through the loop, so as to
+  // Do several (5 or 6) digits each time through the loop, so as to
   // minimize the calls to the very expensive emulated div.
-  var radixToPower = goog.math.Integer.fromNumber(Math.pow(radix, 6));
+  var digitBatchSize = (radix == 36 ? 5 : 6);
```

Comment:
Can you please add a comment explaining why 5 is sometimes used? I.e., the issue with base 36.